Assets/Scripts/Sound/FootstepsSound.cs

```diff
@@ -0,0 +1,85 @@
+using Misc;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Sound
+{
+    [RequireComponent(typeof(Rigidbody))]
+    [RequireComponent(typeof(GroundChecker))]
+    public class FootstepsSound : MonoBehaviour
+    {
+        [Header("Requirements")]
+        [SerializeField] private AudioClip _footstepsSound;
+        [SerializeField] private AudioClip _jumpSound;
+        [SerializeField] private AudioClip _landingSound;
+        [SerializeField] private AudioSource _audioSource;
+
+        private Rigidbody _character;
+        private GroundChecker _groundChecker;
+
+        private bool wasGrounded = false;
```

Comment:
Change name of variable

```suggestion
        private bool _wasGrounded = false;
```